notebooks/python/python-basics.ipynb

```diff
@@ -0,0 +1,873 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # calculate something with your variables - what might we want to calculate with deltaH, deltaS, and temp?
</pre><p>Consider explicitly instructing them to calculate deltaG since you expect them to have the value later when they use the round function.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ESCIP/escip.github.io/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>